keycloak/realm.go

```diff
@@ -46,18 +46,18 @@ type Realm struct {
 }
 
 type SmtpServer struct {
-	StartTls           string `json:"starttls,omitempty"`
-	Auth               string `json:"auth,omitempty"`
-	Port               string `json:"port,omitempty"`
-	Host               string `json:"host,omitempty"`
-	ReplyTo            string `json:"replyTo,omitempty"`
-	ReplyToDisplayName string `json:"replyToDisplayName,omitempty"`
-	From               string `json:"from,omitempty"`
-	FromDisplayName    string `json:"fromDisplayName,omitempty"`
-	EnvelopeFrom       string `json:"envelopeFrom,omitempty"`
-	Ssl                string `json:"ssl,omitempty"`
-	User               string `json:"user,omitempty"`
-	Password           string `json:"password,omitempty"`
+	StartTls           KeycloakBoolQuoted `json:"starttls,omitempty"`
```

Comment:
This is a special type that serializes to a JSON string of `"true"` or `"false"`, which is helpful for parts of the Keycloak API that expect boolean strings.  Under the hood it's a regular boolean that is easier to work with in the provider code.

Reply:
Ok, good to know, I will use that in new development